arangod/RestServer/Metrics.cpp

```diff
@@ -101,9 +101,9 @@ void Counter::store(uint64_t const& n) {
 
 void Counter::toPrometheus(std::string& result) const {
   _b.push();
-  result += "#TYPE " + name() + " counter\n";
+  result += "\n#TYPE " + name() + " counter\n";
   result += "#HELP " + name() + " " + help() + "\n";
-  result += name() + "{" + labels() + "} " + std::to_string(load()) + "\n";
+  result += name() + "{" + labels() + "} " + std::to_string(load());
```

Comment:
this is a bit inconsistent with Gauge<T>::toPrometheus, where some other changes were made, but where the result still ends with a `\n`. Is that intentional?

Reply:
addressed